scripts/dotnet-cli-build/PackageTargets.cs

```diff
@@ -97,6 +97,20 @@ public static BuildTargetResult CopySharedHostLayout(BuildTargetContext c)
                 var destFile = file.Replace(Dirs.Stage2, sharedHostRoot);
                 File.Copy(file, destFile, true);
             }
+            
+            string versionedFxrDir = Path.Combine(sharedHostRoot, "host", "fxr", c.BuildContext.Get<BuildVersion>("BuildVersion").NuGetVersion);
+            Directory.CreateDirectory(versionedFxrDir);
+            foreach (var file in Directory.GetFiles(Dirs.Stage2, "*", SearchOption.TopDirectoryOnly))
+            {
+                if (!Path.GetFileName(file).StartsWith("hostfxr"))
+                {
+                    continue;
+                }
+                
+                var destFile = file.Replace(Dirs.Stage2, versionedFxrDir);
+                File.Copy(file, destFile, true);
```

Comment:
No. Delete the one in the root. How will you avoid the unzip overwrite problem if you keep it in the root? Your unzip should not swallow overwrite errors except for dotnet.exe.